### ISSUE TITLE
Basic user onboarding

### ISSUE THREAD

**Issue by joebudi** [Budibase/budibase#5414]
**Describe the bug**
I, the lazy user I am, did not read the description and did not take note of my password. This is minor, but I think a lot of people will miss it.

Maybe we can improve the UX around this? @AndyWhann, what do you think?

![CleanShot 2022-04-14 at 09 51 23](https://user-images.githubusercontent.com/49767913/163350100-e1712a1c-6ad7-4e75-9849-91d899352e08.png)

**Comment by AndyWhann** [comment]
@joebudi 

Yeah I don't think it's common practice to have the password displayed before the account has been created. The expectation is probably, create the account, then get credentials for that account.

I would remove the password from here, then add that information as a final step after the user has created the account. e.g 

`New user has been added!`
`Use this password below to access this account`

**Comment by joebudi** [comment]
Sounds perfect!
